packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -865,59 +903,60 @@ export class GarbageCollector implements IGarbageCollector {
     }
 
     /**
-     * Validates that all new references are correctly identified and processed. The basic principle for validation is
-     * that we should not have new references in the reference graph (GC data) that have not been notified to the
-     * garbage collector via `referenceAdded`.
-     * We validate that the references in the current reference graph should be a subset of the references in the last
-     * run's reference graph + references since the last run.
+     * Finds outbound references to datastores in the current GC graph that aren't in the previous GC graph and returns
+     * them if they are not added by GC's `addedOutboundReference`.
+     *
+     * In more simple terms:
+     * Missing Explicit References = Current References - Previous References - Explicitly Added References;
+     *
+     * Currently outbound references are references from a parent node to child datastore node
```

Comment:
This line is incorrect. outbound references do not have parent child relationship. Its just from one node to another where a node doesn't have to be a data store.